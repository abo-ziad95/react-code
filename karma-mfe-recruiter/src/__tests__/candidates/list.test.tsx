import React from 'react';
import renderer from 'react-test-renderer';
import Karma from "@hatech/karma-core";
import CandidatesList from "../../views/candidates/list";

it("Candidates List renders correctly", () => {
    const candidatesList = renderer
        .create(<Karma><CandidatesList/></Karma>)
        .toJSON();
    expect(candidatesList).toMatchSnapshot();
});