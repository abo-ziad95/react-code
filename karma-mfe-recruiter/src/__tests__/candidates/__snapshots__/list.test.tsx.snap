// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Candidates List renders correctly 1`] = `
<div
  className="wrapper"
>
  <div
    className="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-3 MuiGrid-justify-xs-center"
  >
    <div
      className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-12 MuiGrid-grid-md-12 MuiGrid-grid-lg-12 MuiGrid-grid-xl-12"
    >
      <nav
        className="MuiTypography-root MuiBreadcrumbs-root MuiTypography-body1 MuiTypography-colorTextSecondary"
      >
        <ol
          className="MuiBreadcrumbs-ol"
        >
          <li
            className="MuiBreadcrumbs-li"
          >
            <a
              className="MuiTypography-root MuiLink-root MuiLink-underlineHover MuiTypography-colorInherit"
              onBlur={[Function]}
              onClick={[Function]}
              onFocus={[Function]}
            >
              Organizations
            </a>
          </li>
          <li
            aria-hidden={true}
            className="PrivateBreadcrumbSeparator-root-144 MuiBreadcrumbs-separator"
          >
            /
          </li>
          <li
            className="MuiBreadcrumbs-li"
          >
            <a
              className="MuiTypography-root MuiLink-root MuiLink-underlineHover MuiTypography-colorInherit"
              onBlur={[Function]}
              onClick={[Function]}
              onFocus={[Function]}
            >
              <div
                className="MuiSkeleton-root MuiSkeleton-text MuiSkeleton-animate"
                style={
                  Object {
                    "height": undefined,
                    "width": undefined,
                  }
                }
              />
            </a>
          </li>
          <li
            aria-hidden={true}
            className="PrivateBreadcrumbSeparator-root-144 MuiBreadcrumbs-separator"
          >
            /
          </li>
          <li
            className="MuiBreadcrumbs-li"
          >
            <a
              className="MuiTypography-root MuiLink-root MuiLink-underlineHover MuiTypography-colorInherit"
              onBlur={[Function]}
              onClick={[Function]}
              onFocus={[Function]}
            >
              Jobs
            </a>
          </li>
          <li
            aria-hidden={true}
            className="PrivateBreadcrumbSeparator-root-144 MuiBreadcrumbs-separator"
          >
            /
          </li>
          <li
            className="MuiBreadcrumbs-li"
          >
            <a
              className="MuiTypography-root MuiLink-root MuiLink-underlineHover MuiTypography-colorInherit"
              onBlur={[Function]}
              onClick={[Function]}
              onFocus={[Function]}
            >
              <div
                className="MuiSkeleton-root MuiSkeleton-text MuiSkeleton-animate"
                style={
                  Object {
                    "height": undefined,
                    "width": undefined,
                  }
                }
              />
            </a>
          </li>
          <li
            aria-hidden={true}
            className="PrivateBreadcrumbSeparator-root-144 MuiBreadcrumbs-separator"
          >
            /
          </li>
          <li
            className="MuiBreadcrumbs-li"
          >
            <p
              className="MuiTypography-root MuiTypography-body1 MuiTypography-colorPrimary"
            >
              Candidates
            </p>
          </li>
        </ol>
      </nav>
    </div>
    <div
      className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-12 MuiGrid-grid-md-12 MuiGrid-grid-lg-12 MuiGrid-grid-xl-12"
    >
      <div
        className="MuiPaper-root MuiPaper-elevation1 MuiCard-root MuiPaper-rounded"
      >
        <div
          className="MuiCardHeader-root"
        >
          <div
            className="MuiCardHeader-content"
          >
            <span
              className="MuiTypography-root MuiCardHeader-title MuiTypography-h5 MuiTypography-displayBlock"
            >
              Candidates
            </span>
          </div>
        </div>
        <hr
          className="MuiDivider-root"
        />
        <ul
          className="MuiList-root MuiList-padding"
        >
          <li
            className="MuiListItem-root MuiListItem-gutters"
            disabled={false}
          >
            <div
              className="MuiListItemText-root"
            >
              <span
                className="MuiTypography-root MuiListItemText-primary MuiTypography-body1"
              >
                No candidates to display
              </span>
            </div>
          </li>
        </ul>
      </div>
    </div>
    <div
      className="MuiGrid-root MuiGrid-item"
      style={
        Object {
          "textAlign": "center",
        }
      }
    >
      <button
        className="MuiButtonBase-root MuiButton-root MuiButton-outlined MuiButton-outlinedPrimary Mui-disabled Mui-disabled"
        disabled={true}
        id="load-more"
        onBlur={[Function]}
        onClick={[Function]}
        onDragLeave={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        onKeyUp={[Function]}
        onMouseDown={[Function]}
        onMouseLeave={[Function]}
        onMouseUp={[Function]}
        onTouchEnd={[Function]}
        onTouchMove={[Function]}
        onTouchStart={[Function]}
        tabIndex={-1}
        type="button"
      >
        <span
          className="MuiButton-label"
        >
          Load More
        </span>
      </button>
    </div>
  </div>
</div>
`;
